scripts/SetupServices.ps1

```diff
@@ -20,7 +20,7 @@ Function New-BasicPki {
 }
 
 Function New-NginxSetup {
-    Write-Header "Configuring Nginx..."
+    Write-Header "Configuring Nginx proxy container..."
```

Comment:
I'll let this one through this time, but I do worry that someone will confuse the "proxy" verbiage here to indicate this is an HTTP Proxy instead of a reverse proxy